### PR TITLE
[IMP] Add filter group by payment mode to invoices

### PR DESCRIPTION
**Pull request by rlizana**
Add group by payment mode in invoices

### CODE REVIEW

File: account_payment_partner/views/account_invoice_view.xml [hunk 1]
```diff
@@ -9,6 +9,18 @@
 <openerp>
 <data>
 
+    <record id="invoice_filter" model="ir.ui.view">
+        <field name="name">account_payment_partner.invoice_filter</field>
+        <field name="model">account.invoice</field>
+        <field name="type">search</field>
```
Comment (pedrobaeza) [inline account_payment_partner/views/account_invoice_view.xml]: Please remove this tag (is deprecated and not needed when inheriting).

**Comment by rlizana** [comment]
Ok, tag removed

**Comment by pedrobaeza** [comment]
You should add the string to the translation template.

**Comment by adrienpeiffer** [comment]
:+1:

File: account_payment_partner/views/account_invoice_view.xml [hunk 2]
```diff
@@ -9,6 +9,17 @@
 <openerp>
 <data>
 
+    <record id="invoice_filter" model="ir.ui.view">
+        <field name="name">account_payment_partner.invoice_filter</field>
+        <field name="model">account.invoice</field>
+        <field name="inherit_id" ref="account.view_account_invoice_filter"/>
+        <field name="arch" type="xml">
+            <filter string="Status" position="after">
+                <filter string="Payment mode" context="{'group_by':'payment_mode_id'}"/>
```
Comment (pedrobaeza) [inline account_payment_partner/views/account_invoice_view.xml:18]: Please put name attribute

**Comment by pedrobaeza** [comment]
I'll merge this one and add the name attribute later on direct commit.

**Pull request merged**